amd_openvx/openvx/api/vx_api.cpp

```diff
@@ -9836,6 +9836,26 @@ VX_API_ENTRY vx_status VX_API_CALL vxQueryTensor(vx_tensor tensor, vx_enum attri
 					}
 				}
 #endif
+				else if (tensor_data_type == VX_TYPE_UINT32)
```

Comment:
We don't need to have an api for host pointer for tensor. Instead, you need to use vxMapTensorPatch() to get the host pointer. Please remove the references to host pointer api